MAUI/Cards/Customization.md

```diff
@@ -0,0 +1,225 @@
+    </Grid>
+</cards:SfCardView>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+	Grid mainStack = new Grid()
+	{
+		Children =
+		{
+			new SfCardView(){
+			Content =  new Grid()
+			{
+				Padding = 20,
+				Children =
+				{
+					new Label(){Text="Wells Fargo", HorizontalOptions=LayoutOptions.Start, TextColor=Colors.White, FontSize=20, FontAttributes=FontAttributes.Bold},
+					new Grid(){
+						Children=
+						{
+							new Image(){Source="cardchip.png", WidthRequest=60, HeightRequest=30, HorizontalOptions=LayoutOptions.Center, VerticalOptions=LayoutOptions.Center},
+							new Label(){Text="Business Elite", FontAttributes=FontAttributes.Bold, TextColor=Colors.White, FontSize=17, HorizontalOptions=LayoutOptions.Center,VerticalOptions=LayoutOptions.Center ,Padding=30 }
+						} },
+					new Label(){HorizontalOptions=LayoutOptions.Start ,VerticalOptions=LayoutOptions.End, Text="Rick Sanchez", FontSize=17, FontAttributes=FontAttributes.Bold, TextColor=Colors.White},
+					new Label(){HorizontalOptions=LayoutOptions.Start,  VerticalOptions=LayoutOptions.End, Text="9 0 5 7    4 0 8 1    2 1 7 5    0 0 5 6", TextColor=Colors.White, Padding=10},
+				}
+			},
+			BackgroundColor = Colors.Brown
+			}
+		}
+	};
+           
+{% endhighlight %}
+
+{% endtabs %}
+
+![Card customization in MAUI SfCardLayout](cardview-customization_images/card_Customization.PNG)
+
+### Indicator customization
+
+Indicators are used to signify or communicate the state or level of something.
+
+{% tabs %} 
+
+{% highlight xaml %}
+
+<cards:SfCardView IndicatorColor="LightGreen" BackgroundColor="LightGoldenrodYellow" HeightRequest="300" IndicatorThickness="12" IndicatorPosition="Left" >
+    <Label  Text="SfCardView" VerticalTextAlignment="Center" HorizontalTextAlignment="Center"/>
+</cards:SfCardView>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+SfCardView cardView = new SfCardView() 
+{     
+    Content = new Label() 
+    { 
+        Text = "SfCardView", 
+        HorizontalTextAlignment = TextAlignment.Center, 
+        VerticalTextAlignment =  TextAlignment.Center
+    },
+    IndicatorThickness = 12,
+    HeightRequest = 300,
+    IndicatorPosition = IndicatorPosition.Left,
+    IndicatorColor = Colors.LightGreen,
+	BackgroundColor = Colors.LightGoldenrodYellow
+};
+           
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Indicator in MAUI SfCardLayout](cardview-customization_images/indicator.png)
+
+### FadeOutOnSwiping
+
+The `FadeOutOnSwiping` option can be enabled when the card view is intended to fade in response to swiping.
+
+{% tabs %} 
+
+{% highlight xaml %}
+
+    <cards:SfCardView FadeOutOnSwiping="true"/>
+ 
+{% endhighlight %}
+
+{% highlight C# %}
+
+SfCardView cardView = new SfCardView()
+{
+  FadeOutOnSwiping = true
+};
+
+{% endhighlight %}
+
+{% endtabs %}
+
+N> This property won't work when adding the SfCardView as a child of SfCardLayout.
+
```

Comment:
N> This property will not work when adding the SfCardView as a child of SfCardLayout.